ci-operator/step-registry/telcov10n/metal-single-node-hub-ztp/verify-hub/telcov10n-metal-single-node-hub-ztp-verify-hub-commands.sh

```diff
@@ -122,15 +122,59 @@ EOF
   run_pytest check_hub_installation
 }
 
+function assert_expected_resources_are_available {
+
+  echo "************ telcov10n Assert the expected resources are available ************"
+
+  set -x
+  for ((attempts = 0 ; attempts <  ${max_attempts:=10} ; attempts++)); do
+    {
+      $oc_hub get no,clusterversion,mcp,co,sc,pv &&
+      $oc_hub get subscriptions.operators,OperatorGroup,pvc -A &&
+      $oc_hub get managedcluster &&
+      set +x &&
+      return ;
+    } ||
+    sleep 1m
+  done
+
+  echo "[FAIL] Not all expected resources are available..."
```

Comment:
Should we report this in the Junit or it's not an actual test?
Same for `console_is_available`

Reply:
Exactly. This is to wait for all PODs start properly and then, run the actual test later in [here](https://github.com/openshift/release/blob/d6394396bb52a9ba63ade9afcd70bac2b8db3990/ci-operator/step-registry/telcov10n/metal-single-node-hub-ztp/verify-hub/telcov10n-metal-single-node-hub-ztp-verify-hub-commands.sh#L86-L92).